perpVects/QFPHelpers.cpp

```diff
@@ -37,19 +37,51 @@ std::ostream& operator<<(std::ostream& os, const unsigned __int128 &i){
   return os;
 }
 
-std::string
-getSortName(sort_t val){
-  switch(val){
-  case lt:
-    return "lt";
-  case gt:
-    return "gt";
-  case bi:
-    return "bi";
-  case def:
-    return "us";
-  default:
-    return "something bad happened, undefined sort type";
+namespace FPHelpers{
+    auto
+  swap_float_int(unsigned int val) -> float{
```

Comment:
Is there a good reason why not to put this in the header file as inline?  Not a big deal, but for such simple functions that don't even do any computations, I don't see a reason to keep them in the source file.